### PR TITLE
Add Kudox Rev2

### PR DESCRIPTION
**Pull request by x1-**
## Description

Add support for Kudox Rev. 2

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Kudox Rev. 2

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kudox/rev2/rev2.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "kudox.h"
```
Comment (fauxpark) [inline keyboards/kudox/rev2/rev2.h]:
```suggestion
```

File: keyboards/kudox/rev2/rev2.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    // // green led on
+    // DDRD |= (1<<5);
+    // PORTD &= ~(1<<5);
+
+    // // orange led on
+    // DDRB |= (1<<0);
+    // PORTB &= ~(1<<0);
+
+	matrix_init_user();
+};
```
Comment (fauxpark) [inline keyboards/kudox/rev2/rev2.c]: This function can be deleted.

File: keyboards/kudox/info.json [hunk 3]
```diff
@@ -1,77 +1,4 @@
 {
```
Comment (fauxpark) [inline keyboards/kudox/info.json]: This file can be deleted.

File: keyboards/kudox/readme.md [hunk 4]
```diff
@@ -5,22 +5,25 @@
...
-make kudox/rev1:default
+make kudox/rev2:default
 ```
 
+
 Example of flashing this keyboard:
 
 ```sh
-make kudox/rev1:default:avrdude
+make kudox/rev2:default:avrdude
```
Comment (drashna) [inline keyboards/kudox/readme.md]:
```suggestion
make kudox/rev2:default:flash
```

File: keyboards/kudox/rev2/rev2.c (hunk 2 above)
Comment (drashna) [reply keyboards/kudox/rev2/rev2.c]: Actually, everything below the `include` could be safely removed.

**Comment by x1-** [comment]
@fauxpark @drashna Thank you for reviewing!
I have corrected your suggestions.

**Pull request merged**